src/bodywork/cli/secrets.py

```diff
@@ -133,27 +134,25 @@ def display_secrets(
         this name, defaults to None
     """
     if not k8s.namespace_exists(namespace):
-        print(f"namespace={namespace} could not be found on k8s cluster")
+        print_warn(f"Could not find namespace={namespace} on k8s cluster.")
         return None
+    secrets = k8s.list_secrets(namespace, group)
     if secret_name and not group:
-        print("please specify which secrets group the secret belongs to.")
+        print_warn("Please specify which secrets group the secret belongs to.")
         return None
+    elif secret_name and group:
+        try:
+            complete_secret_name = _create_complete_secret_name(group, secret_name)
+            print_dict(secrets[complete_secret_name].data, complete_secret_name)
```

Comment:
That's nice that we now have a standardised way of printing a Dictionary of values.

Reply:
Yeah - makes it so much easier!